tests/surrogate_test.py

```diff
@@ -180,6 +180,53 @@ def test_surrogate_1():
     assert numpy.allclose(surrogate_values, exact_values)
 
 
+def test_surrogate_1_multi_input():
+    """Test if surrogate generates exact results when call has >1 input."""
+    grid_gen = SmolyakGridGenerator(ChebyshevFirstKind.make_nested_set(2))
+    surrogate = Surrogate([(-1, 1), (-1, 1)], grid_gen)
+    surrogate.train(function_1)
+    # random points in the domain
+    points = [(0.649, -0.9), (-0.885, 1)]
+    surrogate_values = [surrogate(x) for x in points]
+    surrogate_values_multi = surrogate(points)
+    assert numpy.allclose(surrogate_values, surrogate_values_multi)
+
+
+def test_surrogate_1_multi_grid():
+    """Test if surrogate generates exact results if call has >1 input."""
+    grid_gen = SmolyakGridGenerator(ChebyshevFirstKind.make_nested_set(2))
+    surrogate = Surrogate([(-1, 1), (-1, 1)], grid_gen)
+    surrogate.train(function_1)
+    # random points in the domain
+    points = numpy.random.rand(3,4,2)*2 - 1
```

Comment:
Use of `numpy.random` in tests is discouraged because it makes them not reproducible.

If you want to do this, make sure you seed it each time. Using the modern NumPy workflow for creating a RNG will help with this, rather than relying on `numpy.random`. See the docs there or ask if you have questions.